ui/templates/main.html

```diff
@@ -59,7 +59,7 @@ <h1>Φορτίο (fortio) v{{.Version}}{{if not .DoLoad}} control UI{{end}}</h1>
     <br />
     Payload:<br /><textarea name="payload" rows="5" cols="75" id="payload"></textarea><br />
     Load using:<br />
-    tcp/udp/http: <input type="radio" name="runner" value="http" checked/>
+    tcp/udp/http: <input type="radio" name="runner" value="http/tcp/udp" checked/>
```

Comment:
So logs show:
```
13:56:32 I uihandler.go:113> Starting http/tcp/udp load request from [::1]:58518 for udp://localhost:8078
13:56:32 I uihandler.go:182> New run id 1
13:56:32 I udprunner.go:225> Starting udp test for udp://localhost:8078 with 8 threads at 100.0 qps
```

grpc is string value it checked for, everything else depends on url